samples/sensor/qdec/src/main.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Valerio Setti <valerio.setti@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/sys/printk.h>
+
+void main(void)
+{
+	struct sensor_value val;
+	int rc;
+	const struct device *dev = DEVICE_DT_GET(DT_ALIAS(qdec0));
+
+	if (!device_is_ready(dev)) {
+		printk("Qdec device is not ready\n");
+		return;
+	}
+
+	printk("Quadrature decoder sensor test\n");
```

Comment:
see my comment below

Reply:
Yes, once we solve that comment, I'll set this accordingly